frontend/server/cmd/APITool.php

```diff
@@ -1214,6 +1214,30 @@ public function generateDocumentation(): void {
         }
     }
 
+    public function generateControllersDoc(): void {
+        $readmeUrl = 'https://github.com/omegaup/omegaup/blob/main/frontend/server/src/Controllers/README.md';
+        echo "## API Controllers\n\n";
+        echo "For more information about the API controllers, please refer to the [Controllers README]({$readmeUrl}).\n\n";
+        echo "### Index\n\n";
+        ksort($this->controllers);
+        foreach ($this->controllers as $controller) {
+            echo (
+                "- [{$controller->classBasename}](#" .
```

Comment:
Acá falta indicar la URL del readme para que el nombre del controller también se convierta en un link válido:
```suggestion
                "- [{$controller->classBasename}](#{$readmeUrl}" .
```